WooCommerce/Classes/Authentication/Application Password/ApplicationPasswordAuthorizationWebViewController.swift

```diff
@@ -157,20 +163,46 @@ private extension ApplicationPasswordAuthorizationWebViewController {
                 guard let url = try await viewModel.fetchAuthURL() else {
                     DDLogError("⛔️ No authorization URL found for application passwords")
                     analytics.track(.applicationPasswordAuthorizationURLNotAvailable)
-                    return showErrorAlert(message: Localization.applicationPasswordDisabled)
+                    navigateToApplicationPasswordDisabledUI()
+                    return
                 }
                 loadAuthorizationPage(url: url)
             } catch {
                 DDLogError("⛔️ Error fetching authorization URL for application passwords \(error)")
                 analytics.track(.applicationPasswordAuthorizationURLFetchFailed, withError: error)
-                showErrorAlert(message: Localization.errorFetchingAuthURL, onRetry: { [weak self] in
-                    self?.fetchAuthorizationURL()
-                })
+                if let authError = error as? Networking.RequestAuthenticatorError,
+                   authError == .applicationPasswordNotAvailable {
+                    navigateToApplicationPasswordDisabledUI()
+                } else {
+                    showErrorAlert(message: Localization.authorizationRejected)
```

Comment:
We should display `errorFetchingAuthURL` instead of `authorizationRejected` here. You can restore `errorFetchingAuthURL` from the git history.